src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -1422,7 +1434,12 @@ File {} line {}.)", wname, location.keyword, location.filename, location.lineno)
 
                 auto injection = std::make_shared<Well::WellInjectionProperties>(well2.getInjectionProperties());
                 auto previousInjectorType = injection->injectorType;
-                injection->handleWCONINJE(record, well2.isAvailableForGroupControl(), well_name);
+                const auto& u = this->m_static.m_unit_system;
+                const double bhp = u.from_si(UnitSystem::measure::pressure,
+                                             this->m_static.default_bhp_limit_inj);
```

Comment:
It's certainly inferred from the member name, but I might probably name this `bhp` object `default_bhp_limit` or something along those lines to be very explicit about its purpose.

Reply:
sure, my 80char ocd is likely not the priority here ;)